index.html

```diff
@@ -2905,6 +2915,168 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+A Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls or a <a>holder</a>
+has been designated to use one or more particular confirmation methods and to
```

Comment:
```suggestion
credential</a> to declare that the <a>subject</a> controls, or a <a>holder</a>
has been designated to use, one or more particular confirmation methods and to
```

Not sure if this is an improvement, but this sentence felt clunky to read as it currently is.

Reply:
I also find the last part hard to follow. "and to bind the claims ..." Maybe this could be a separate sentence